configs/setups/awiesm/awiesm-2.1.yaml

```
@@ -210,6 +211,10 @@ fesom:
                 "2.1":
                         version: "2.1"
                         ALE_scheme: 'linfs' # NOTE(PG): This should probably be overriden in the AWIESM 2.x yaml or in the user runscript.
+                "2.1-paleodyn-gamma":
+                        # FIXME(PG): ....does that make sense? Choose version sets the version???
+                        version: "2.1-paleodyn-gamma"
+                        ALE_scheme: 'zstar' # NOTE(PG): This should probably be overriden in the AWIESM 2.x yaml or in the user runscript.
```

```suggestion
                        ALE_scheme: 'zstar'
```
The `ALE_scheme` is now in the correct file so I think we can get rid of this comment.